src/Integration/Monolog/CloudWatch/src/CloudWatchLogsHandler.php

```diff
@@ -252,4 +271,39 @@ private function send(array $entries): void
 
         $this->client->putLogEvents($data);
     }
+
+    /**
+     * @param LogRecord|array $record
+     */
+    public function handle($record): bool
+    {
+        if(is_array($record)){
+            $message = $record['message']??null;
+            $channel = $record['channel']??null;;
+            $level= $record['level']? Level::fromValue($record['level']) : null;;
+            $context= $record['context']??null;;
+            $extra= $record['extra']??null;;
+            $datetime= $record['datetime'] ? \DateTimeImmutable::createFromMutable($record['datetime']) : null;;
+            $formatted= $record['formatted']??null;;
+            $record = new LogRecord($datetime, $channel, $level, $message, $context, $extra, $formatted);
```

Comment:
This class does not exist in monolog 2

Reply:
trying to call it only when we use monolog 3 by the if statement check